src/Files.App/Helpers/Win32/Win32Helper.Process.cs

```diff
@@ -37,5 +37,76 @@ public static async Task<bool> InvokeWin32ComponentsAsync(IEnumerable<string> ap
 				return await LaunchHelper.LaunchAppAsync(application, arguments, workingDirectory);
 			}
 		}
+
+		/// <summary>
+		/// Gets process(es) that have a lock on the specified file.
+		/// </summary>
+		/// <param name="path">Path of the file.</param>
+		/// <returns>Processes locking the file.</returns>
+		/// <remarks>
+		/// For more info, visit
+		/// <br/>
+		/// - <a href="https://learn.microsoft.com/ja-jp/windows/win32/api/restartmanager/nf-restartmanager-rmgetlist"/>
+		/// <br/>
+		/// - <a href="https://stackoverflow.com/questions/317071/how-do-i-find-out-which-process-is-locking-a-file-using-net/317209#317209"/>
+		/// </remarks>
+		public static List<Process> WhoIsLocking(string[] resources)
```

Comment:
This has been inside of FileUtils, which has been interop helper.
So, this should've been inside of a normal helper.